apps/wasm/mvp/readme.md

```diff
@@ -0,0 +1,75 @@
+# Prerequisites
+
+[rust stable](https://rustup.rs/)
+
+[nodejs lts](https://nodejs.org/en/download/)
+
+[go 1.17.x](https://go.dev/dl/)
+
+[yarn](https://classic.yarnpkg.com/en/docs/install)
+
+```
+cargo install wasm-pack
+```
+
+# Commands
+
+- Start [proxy node](https://github.com/ChainSafe/mina-rs/tree/main/apps/mvp-proxy)
+
+  ```
+  cd apps/mvp-proxy
+  go mod vendor
+  go run .
+  ```
+
+- Build and Run
+
+  ```
+  cd apps/wasm/mvp
+  yarn
```

Comment:
I had issues here with incompatible node versions. Might be worth mentioning that it required node 16.0 or later

Reply:
Thx for pointing it out, I haven't tried node  version 14 and prior, a little surprised to know 14 not working. Will update the doc